mobilitypath_publisher/config/parameters.yaml

```diff
@@ -0,0 +1,3 @@
+# uncertainty threshould for updating the mobility path trajectory
+x_threshold: 20.0; #m
+y_threshold: 3; #m (width of a lane)
```

Comment:
Remove ";" from both lines. Does this actually load? I'm concerned this node hasn't actually been run. I think semi colons in the yaml would cause a failure.

Reply:
changed